src/app/components/modal.tsx

```diff
@@ -8,9 +8,11 @@ interface ModalProps {
   contentClass?: string;
   containerClass?: string;
   closeButtonClass?: string;
+  redirectRoute: string;
 }
 
 export function Modal({
+  redirectRoute,
```

Comment:
added `redirectRoute`, so modal component can be used for redirecting to other routes